cli/src/commands/help.rs

```diff
@@ -42,8 +55,70 @@ pub(crate) fn cmd_help(
         .app()
         .clone()
         .subcommand_required(true)
+        .after_help(format_categories(command.app()))
```

Comment:
I would move this to `enum Command` attribute so `jj --help` shows the identical content.